pkg/base/replicated_test.go

```diff
@@ -588,14 +592,84 @@ metadata:
   labels:
     app.kubernetes.io/name: multidoc`),
 			},
+			expectedKotsKinds: map[string][]byte{
+				"config.yaml": []byte(`apiVersion: kots.io/v1beta1
+kind: Config
+metadata:
+  creationTimestamp: null
+  name: config-sample
+spec:
+  groups:
+  - description: info for pod
+    items:
+    - default: test
+      name: podName
+      type: text
+      value: testPod
+    - default: /var/www/html
+      name: mountPath
+      type: text
+      value: ""
+    name: podInfo
+    title: ""
+  - description: Buncha Secrets
+    items:
+    - countByGroup:
+        secrets: 3
+      default: onetwothree
+      minimumCount: 1
+      name: secretName
+      repeatable: true
+      templates:
+      - apiVersion: apps/v1
+        kind: Deployment
+        name: my-deploy
+        namespace: my-app
+        yamlPath: spec.template.spec.volumes[1].projected.sources[1]
+      - apiVersion: v1
+        kind: Secret
+        name: my-secret
+        namespace: my-app
+      title: Secret Name
+      type: text
+      value: ""
+      valuesByGroup:
+        secrets:
+          secretName-1: MTIz
+          secretName-2: MTIz
+          secretName-3: MTIz
+    name: secrets
+    title: Secrets
+status: {}
+`,
+				),
+				"userdata/configvalues.yaml": []byte(`apiVersion: kots.io/v1beta1
```

Comment:
i thought that the config values are written at the top level directory as `configvalues.yaml` https://github.com/replicatedhq/kots/pull/3767/files#diff-b8cf2cbeb02e811d9f2e22ca6e1a73949feb7d771328121b1d1b4404007631d6R190

why are they under `userdata/` here?

Reply:
The tests actually convert the raw data to a KotsKind struct for comparison to avoid issues like different yaml list ordering, so these filenames are ignored, but I updated them for consistency